tests/baselines/reference/tsc/incremental/change-to-modifier-of-class-expression-field-with-declaration-emit-enabled.js

```diff
@@ -241,11 +241,11 @@ exitCode:: ExitStatus.DiagnosticsPresent_OutputsSkipped
       "./messageableperson.ts": {
         "original": {
           "version": "3462418372-const Messageable = () => {\n    return class MessageableClass {\n        protected message = 'hello';\n    }\n};\nconst wrapper = () => Messageable();\ntype MessageablePerson = InstanceType<ReturnType<typeof wrapper>>;\nexport default MessageablePerson;",
-          "signature": "-21450256696-declare const wrapper: () => {\n    new (): {\n        message: string;\n    };\n};\ntype MessageablePerson = InstanceType<ReturnType<typeof wrapper>>;\nexport default MessageablePerson;\n(116,7)Error4094: Property 'message' of exported class expression may not be private or protected.",
+          "signature": "-2474549887-export default MessageablePerson;\n",
```

Comment:
This is the main issue: This one doesnt report error so we are not able to know that there is no change in signature, and we would need to rebuild.